FiducialRegistrationWizard/MRML/vtkMRMLFiducialRegistrationWizardNode.cxx

```diff
@@ -319,6 +320,56 @@ int vtkMRMLFiducialRegistrationWizardNode::UpdateModeFromString( std::string upd
 }
 
 //------------------------------------------------------------------------------
+void vtkMRMLFiducialRegistrationWizardNode::SetInputFormat( int newInputFormat )
+{
+  if ( this->GetInputFormat() == newInputFormat )
+  {
+    // no change
+    return;
+  }
+  this->InputFormat = newInputFormat;
+  this->Modified();
+  this->InvokeCustomModifiedEvent(InputDataModifiedEvent);
+}
+
+//------------------------------------------------------------------------------
+std::string vtkMRMLFiducialRegistrationWizardNode::InputFormatAsString( int inputFormat )
+{
+  switch ( inputFormat )
+  {
+    case INPUT_FORMAT_ORDERED_PAIRS:
+    {
+      return "Ordered Pairs";
```

Comment:
I don't understand, and I don't see tr() used in many places in SlicerIGT. What is the problem that it is supposed to solve? Is it the space in "Ordered Pairs" when writing to XML...?

Where should I use the tr() function?